btcstaking/staking_test.go

```diff
@@ -151,11 +151,14 @@ func testSlashingTx(
 	slashingAddress, err := genRandomBTCAddress(r)
 	require.NoError(t, err)
 
+	slashingPkScript, err := txscript.PayToAddrScript(slashingAddress)
+	require.NoError(t, err)
```

Comment:
so the pros of using it are described in here - https://github.com/babylonlabs-io/pm/blob/main/rfc/rfc-003-btc-staking-parameters.md#change-slashing_address-to-slashing_pk_script

Ultimately it is even better to have it as `[]byte` than `string` as then when validating message babylon node can just do `bytes.equal` without any kind of decoding.

So tldr:
- it is less work for Babylon node
- we have more flexibility in terms of defining slashing address. Now it can by `OP_RETURN` without any bytes, which is pure burn address

Reply:
so this is bit will be implemented in subsequent pr's mainly handling the upgrade. 

The problem is, now we already have genesis params fixed. So what I want to do is to:
- create new params structure in the code 
- overwrite the old params with old structure with new params with new structure.

This implies that new params will actually be hard coded and not read from `genesis.json` (example how osmosis hardcode some of the params upgrade:https://github.com/osmosis-labs/osmosis/blob/main/app/upgrades/v25/upgrades.go#L94)